graphite-ch-optimizer.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/spf13/viper"
 )
 
+const version = "1.0.0"
```

Comment:
i have one suggestion you might think about: make this const a variable and assign it something like "dev" as default. then when you build the binary you can override the variable with `-ldflags "-X main.version=$(VERSION)"` in your makefile. that way you have the possibility to pass anything you want, like the git tag or any other version. this would make it more flexible